rust/lance-encoding/src/encodings/physical/bitpacking.rs

```diff
@@ -318,28 +314,46 @@ fn bitpack_out_of_line<T: ArrowNativeType + BitPacking>(
         return LanceBuffer::reinterpret_vec(output);
     }
 
-    // If we get here then the last chunk needs to be padded with zeros
-    let remaining_items = data_buffer.len() % ELEMS_PER_CHUNK as usize;
     let last_chunk_start = num_whole_chunks * ELEMS_PER_CHUNK as usize;
+    output.truncate(num_whole_chunks * words_per_chunk);
```

Comment:
Minor nit: instead of truncating here maybe we just set the length with `num_whole_chunks` above?

Reply:
Yep, it should work.